scipy/stats/mstats_basic.py

```diff
@@ -2218,6 +2219,88 @@ def kurtosis(a, axis=0, fisher=True, bias=True):
         return vals
 
 
+def gstd(a, axis=0, ddof=1):
+    """Calculate the geometric standard deviation of an array
+
+    The geometric standard deviations is defined as the exponent of the
+    standard deviation of ``log(a)``. Mathematically the population
+    geometric standard deviation is evaluated as::
+
+        gstd = exp(std(log(a))
+
+    The geometric standard deviation describes the spread of a set of numbers
+    where :func: `gmean` is the preferred mean. It is a multiplicative
```

Comment:
`:func:` is extraneous here, single backticks do it automatically.

Reply:
Thanks for picking it up, I'll remove it in the next commit.